scripts/triggered_mails.py

```diff
@@ -27,7 +26,7 @@ def main(dry_run: bool = True):
         logger.info('No users matched inactivity criteria.')
         return
 
-    for user in users.iterator():
+    for user in users.all():
```

Comment:
Is there a limitation for us not to be able to use `.iterator()`.

* `.all()` is very expensive if there is a lot of user.
* I don't think we reuse the set of users, so no need to use `.all()`

Reply:
You are right, we can just do `for user in users:` ...